include/variables.h

```diff
@@ -2512,10 +2509,10 @@ extern IrqMgrClient irqClient;
 extern OSMesgQueue irqMgrMsgQ;
 extern OSMesg irqMgrMsgBuf[60];
 extern OSThread sGraphThread;
-extern u8 sGraphStack[6144];
-extern u8 schedStack[1536];
-extern u8 audioStack[2048];
-extern u8 padmgrStack[1280];
+extern STACK(sGraphStack, 0x1800);
+extern STACK(schedStack, 0x600);
+extern STACK(audioStack, 0x800);
+extern STACK(padmgrStack, 0x500);
 extern StackEntry sGraphStackInfo;
 extern StackEntry schedStackEntry;
 extern StackEntry audioStackEntry;
```

Comment:
```suggestion
extern StackEntry sSchedInfo;
extern StackEntry sAudioInfo;
```
Also `padmgrStackEntry` -> `sPadMgrInfo`